packages/fxa-content-server/app/tests/spec/views/base.js

```diff
@@ -424,8 +424,8 @@ describe('views/base', function() {
       view.rerender();
 
       return P.delay(100).then(() => {
-        assert.isTrue(view.render.calledOnce);
-        assert.isTrue(view.afterVisible.calledOnce);
+        assert.equal(view.render.callCount, 1);
+        assert.equal(view.afterVisible.callCount, 1);
```

Comment:
Tweaked for more helpful assertion failures, so I could see what the call count was in the error message rather than "expected false to be true".